### ISSUE TITLE
No resource identifier zxing_framing_rect

### ISSUE THREAD

**Issue by EphraimKigamba** [journeyapps/zxing-android-embedded#103]
Can someone help me out.

I have added zxing core library and zxing-android-embedded library to my application. My aim is to create an activity which enables continuous scanning using the component CompoundBarcodeView

Here is part of my gradle file:

```gradle
dependencies {
    compile fileTree(include: ['*.jar'], dir: 'libs')
    //6.5.87'
    //compile 'com.android.support:appcompat-v7:21.0.2'
    compile project(':fab_library')
    compile project(':kigamba_libs')
    compile project(':lib_bootstrap2')
    compile project(':library_material_ripple_master')
    //compile 'com.afollestad:material-dialogs:0.7.7.0'
    compile 'com.google.android.gms:play-services:8.1.0'
    //compile 'com.android.support:appcompat-v7:22.1.1'
    compile 'com.android.support:appcompat-v7:21.0.2'
    compile 'com.google.android.gms:play-services-ads:8.1.0'
    compile 'com.google.android.gms:play-services-identity:8.1.0'
    compile 'com.google.android.gms:play-services-gcm:8.1.0'
    compile 'com.journeyapps:zxing-android-embedded:3.0.3@aar'
    compile 'com.google.zxing:core:3.2.0'
}
```

Here is the layout custom_barcode_scanner.xml

```xml
<?xml version="1.0" encoding="utf-8"?>
<merge xmlns:android="http://schemas.android.com/apk/res/android"
       xmlns:tools="http://schemas.android.com/tools"
       xmlns:app="http://schemas.android.com/apk/res-auto">

    <com.journeyapps.barcodescanner.BarcodeView
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:id="@+id/zxing_barcode_surface"
        app:zxing_framing_rect_width="250dp"
        app:zxing_framing_rect_height="50dp"/>

    <com.journeyapps.barcodescanner.ViewfinderView
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:id="@+id/zxing_viewfinder_view"/>

    <TextView
        android:id="@+id/zxing_status_view"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_gravity="bottom|center_horizontal"
        android:background="@color/zxing_transparent"
        android:text="@string/zxing_msg_default_status"
        android:textColor="@color/zxing_status_text"/>

</merge>
```

Here is layout file activity_scanning.xml

```xml
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="match_parent" android:paddingLeft="@dimen/activity_horizontal_margin"
    android:paddingRight="@dimen/activity_horizontal_margin"
    android:paddingTop="@dimen/activity_vertical_margin"
    android:paddingBottom="@dimen/activity_vertical_margin">

    <LinearLayout
        android:orientation="vertical"
        android:layout_width="match_parent"
        android:layout_height="match_parent">

        <com.journeyapps.barcodescanner.CompoundBarcodeView
            android:id="@+id/zxing_barcode_scanner"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            app:zxing_scanner_layout="@layout/custom_barcode_scanner">
        </com.journeyapps.barcodescanner.CompoundBarcodeView>

    </LinearLayout>

    <Button
        android:id="@+id/switch_flashlight"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:text="ON"
        android:layout_alignParentTop="true"
        android:layout_centerHorizontal="true"
        android:onClick="switchFlashlight"/>
</RelativeLayout>
```

Whenever I try to run the application I get the error that the attribute zxing_framing_rect_width cannot be found.

Could someone help me out

**Comment by rkistner** [comment]
Are you getting a compile error or runtime error? Can you post the complete details of the error?

**Comment by EphraimKigamba** [comment]
Yea, It's a compile time error.

Let me rewrite the code and post it again

**Comment by chenluke** [comment]
Is that fixed? I have the same problem. Thanks!

**Comment by chenluke** [comment]
I found out that your "com.journeyapps:zxing-android-embedded:3.0.3@aar" doesn't contain framing rect size code. Please double check.

**Comment by EphraimKigamba** [comment]
I actually found the framing_rect_size code myself since I tried both adding the aar dependency in my gradle file and adding the source code as a library but for some reason, it not found

**Comment by rkistner** [comment]
Can you post a sample project where this happens, as well as the full error where this happens? It works in the sample app in this repo.

**Comment by EphraimKigamba** [comment]
Ok.

**Comment by EphraimKigamba** [comment]
![image](https://cloud.githubusercontent.com/assets/9572675/11405223/85268f10-93b7-11e5-8472-552229bb0ee1.png)

**Comment by EphraimKigamba** [comment]
There is the error. It is a compile-time error

**Comment by ivankolev** [comment]
Same problem here. I think it has to do with the schema definitions in the xml files, I have replaced the res-auto one with xmlns:app="http://schemas.android.com/apk/lib/com.journeyapps.barcodescanner" and it compiles fine now. I have other problem though, the custom activity styling does not seem to be applied, no matter what values I enter for the barcodeView width and height it stays square

**Comment by chenluke** [comment]
@rkistner I created a demo project for you to show this error.
https://github.com/chenluke/barcode_demo

**Comment by rkistner** [comment]
Thanks for all the info. I can reproduce it now with the demo from @chenluke, will look into this soon.

**Comment by EphraimKigamba** [comment]
Thanks @ivankolev for the tip. Works like a charm now. Not yet tried the styling for the custom activity. Just started

**Comment by radityagumay** [comment]
Hi @rkistner i have same problem

**Comment by Paschmid** [comment]
I have the same problem. @rkistner did you already have a look at it? I'd like to include the scanner with a customized layout into an app soon. I could try to fix it if you point me in the right direction.

**Comment by EphraimKigamba** [comment]
Hey Paschmid, ivan Kolev's solution works

**Comment by EphraimKigamba** [comment]
Try it out

**Comment by Paschmid** [comment]
I did try it out and it compiles, yes. But as Ivan Kolev said, the customization of the barcodeView doesn't work anymore and this is important to me since I have small barcodes that I want to detect.

**Comment by rkistner** [comment]
Fixed in 3.1.0.
